web/controllers/consent_controller.ex

```diff
@@ -14,8 +14,23 @@ defmodule ResearchResource.ConsentController do
 
   # save user and consent
   def create(conn, %{"consent" => consent}) do
+    case check_consent(consent) do
+      true ->
+        create_user(conn, consent)
```

Comment:
Could we break the `true` case down into smaller functions, something like:
```elixir
true ->
  conn
  |> consent_to_record()
  |> redcap_save_record()
  |> qualtrics_contact()
  |> update_repo()
  |> assign(:current_user, user)
  |> redirect(to: qualtrics_path(conn, :new))
```

I would find reading that easier to see what is going on